pom.xml

```
@@ -473,7 +473,8 @@
             <bnd><![CDATA[
               Bundle-SymbolicName: ${Bundle-SymbolicName}
               # Export packages not containing the substring 'internal'
-              -exportcontents: ${removeall;${packages};${packages;NAMED;*internal*}}
+              -exportcontents: *.internal.*;x-internal:=true, \
```

Should `x-internal` also be added to impl packages?
[api-compatibility/package-level-contracts](https://maven.apache.org/resolver/api-compatibility.html#package-level-contracts) says both kind of packages are to be treated as 'non-public'?
```suggestion
              -exportcontents: *.internal.*;x-internal:=true, \
                *.impl.*;x-internal:=true, \
```

---

Yes I think so.